lib/helpers/backend/valid-isbn.ts

```diff
@@ -0,0 +1,4 @@
+export function isValidISBN(isbn: string) {
+  if (isbn.length == 13 && isbn.match(/^[0-9]+$/)) return true;
```

Comment:
You could do `return isbn.length == 13 && isbn.match(/^[0-9]+$/)`. Not a big deal, though.

Reply:
Will do.